<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>104</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>12</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>16</width>
    <height>20</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>ClinicM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/1329798553_doctor.png</normaloff>:/new/1329798553_doctor.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>1022</width>
     <height>220</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAgenda"/>
   <addaction name="separator"/>
   <addaction name="actionPaciente"/>
   <addaction name="separator"/>
   <addaction name="actionHistoriaClinica"/>
   <addaction name="separator"/>
   <addaction name="actionUsuarios"/>
   <addaction name="separator"/>
   <addaction name="actionAlertas"/>
   <addaction name="separator"/>
   <addaction name="actionPersonal"/>
   <addaction name="separator"/>
   <addaction name="actionCaja"/>
   <addaction name="separator"/>
   <addaction name="actionMailing"/>
   <addaction name="separator"/>
   <addaction name="actionFacebook"/>
  </widget>
  <action name="actionAgenda">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339085_ebooks.png</normaloff>:/new/1354339085_ebooks.png</iconset>
   </property>
   <property name="text">
    <string>Agenda</string>
   </property>
   <property name="iconText">
    <string>&amp;Agenda</string>
   </property>
   <property name="toolTip">
    <string>Agenda</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPaciente">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339048_people-b.png</normaloff>:/new/1354339048_people-b.png</iconset>
   </property>
   <property name="text">
    <string>Paciente</string>
   </property>
   <property name="iconText">
    <string>&amp;Paciente</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionHistoriaClinica">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339039_checklist_pencil-b.png</normaloff>:/new/1354339039_checklist_pencil-b.png</iconset>
   </property>
   <property name="text">
    <string>Historia Clínica</string>
   </property>
   <property name="iconText">
    <string>&amp;Historia Clinica</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339027_people_stacked-b.png</normaloff>:/new/1354339027_people_stacked-b.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
   <property name="iconText">
    <string>&amp;Usuarios</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionMailing">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339053_globe-b.png</normaloff>:/new/1354339053_globe-b.png</iconset>
   </property>
   <property name="text">
    <string>Mailing</string>
   </property>
   <property name="iconText">
    <string>Página &amp;Web</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionAlertas">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339097_localbusiness-y.png</normaloff>:/new/1354339097_localbusiness-y.png</iconset>
   </property>
   <property name="text">
    <string>Alertas</string>
   </property>
   <property name="iconText">
    <string>Al&amp;ertas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionFacebook">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/facebook (1).png</normaloff>:/new/facebook (1).png</iconset>
   </property>
   <property name="text">
    <string>acebook</string>
   </property>
   <property name="iconText">
    <string>&amp;Facebook</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionPersonal">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/1354339051_profiles-o.png</normaloff>:/new/1354339051_profiles-o.png</iconset>
   </property>
   <property name="text">
    <string>Personal</string>
   </property>
   <property name="iconText">
    <string>Per&amp;sonal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTratamientos">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/Stethoscope_Black.png</normaloff>:/new/Stethoscope_Black.png</iconset>
   </property>
   <property name="text">
    <string>Tratamientos</string>
   </property>
   <property name="iconText">
    <string>&amp;Tratamientos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCaja">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/cash_register.png</normaloff>:/new/cash_register.png</iconset>
   </property>
   <property name="text">
    <string>Caja</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
